qiita_pet/handlers/study_handlers/tests/test_processing.py

```diff
@@ -0,0 +1,28 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from unittest import main
+
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+
+class ProcessArtifactHandler(TestHandlerBase):
+    # TODO: Missing tests
```

Comment:
Same answer as above - want first to be able to pass the tests and then will finish later - note that this is going to a branch. Will lis them in the todo list.